### PR TITLE
feat: check for aws credentials in the enviornment variables

### PR DESCRIPTION
**Pull request by rrijkse**
# Description

Adds a feature which checks for existing AWS credentials in the Environment Variables and both prints a warning for the users to see and will unset the variable for the remainder of the job so they do not interfere. 

This checks for the existence of VAULT_* variables to ensure they are set correctly and exits with a nice error message if they do not.

This also adds testing for the Orb itself.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by infradannii — DISMISSED** [review]
Looks good! 👍

**Review by infradannii — DISMISSED** [review]

File: src/commands/get-aws-creds.yml [hunk 1]
```diff
@@ -54,7 +82,8 @@ steps:
         done
 
         if ((retries == 0 )); then
-          echo "The credentials never started working even after 1 min, so exiting..."
+          echo "The AWS Credentials never started working even after 1 min, so exiting..."
+          echo "Make sure you have"
```
Comment (infradannii) [inline src/commands/get-aws-creds.yml]: Looks incomplete..

File: src/commands/login.yml [hunk 2]
```diff
@@ -15,10 +15,39 @@ parameters:
...
+        echo
+        if [[ -n "$ERROR" ]]; then
+          echo "ERROR:"
+          echo
+          echo "You should check your projects or context Environment Variables page to see if you set <<parameters.vault-addr>>,"
+          echo "<<parameters.vault-role-id>> and <<parameters.vault-secret-id>>"
+          echo
+          exit 1
+        else
+          echo "Environment variables look all good."
```
Comment (infradannii) [inline src/commands/login.yml]: Nit: here we have "look all good." and above "look all clean."
I think we should have consistent messages.
E.g. "environment variables passed initial check"

File: src/commands/get-aws-creds.yml (hunk 1 above)
Comment (rrijkse) [reply src/commands/get-aws-creds.yml]: Sure is.... have fixed it.

File: src/commands/login.yml (hunk 2 above)
Comment (rrijkse) [reply src/commands/login.yml]: How about what I changed it to? They are two different checks, one checks if the variables are not there and the other makes sure they are.

Comment (infradannii) [reply src/commands/login.yml]: 👍 looks great!

**Review by infradannii — APPROVED** [review]
All looks good!

**Pull request merged**